### PR TITLE
Update README to include the project scope

### PR DESCRIPTION
**Pull request by yungyuc**
Add project scope in the README file.  We can use this PR to discuss what is missing or unclear.

Also see the [previous discussion](https://github.com/solvcon/seminar/discussions/10).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,28 @@
+**UNDER DEVELOPMENT**
```
Comment (yungyuc) [inline README.md:1]: I'd like to add this disclaimer before we come up with a plan for a release.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
 ## Usage
 
 By invoking this one command should bring you the whole SOLVCON stack:
 
-```
+```bash
 ./devenv.sh
 ```
+
+<!-- vim: set ff=unix ft=markdown fenc=utf8 sw=2 tw=79: -->
```
Comment (yungyuc) [inline README.md:28]: Using fixed width for lines will make paragraphs obvious in plain text.  I think it's good.  Any comments or concerns?

File: README.md [hunk 3]
```diff
@@ -1,12 +1,28 @@
+**UNDER DEVELOPMENT**
+
 # SOLVCON Development Environment
 
-This repository collects the scripts to build your own Solvcon development environment locally with all necessary third-party libraries and the corresponding source code. That means a whole SOLVCON stack and their debugging context will be delivered by using this tool.
+This project provides a reference runtime environment for developing SOLVCON
+and related projects to reduce unnecessary misunderstandings.  It supports the
+following features:
+
+1. Support ubuntu and macOS.
```
Comment (yungyuc) [inline README.md:9]: Clarification and discussion about the requirements are welcomed.

**Comment by yungyuc** [comment]
@tai271828 @j8xixo12 I added some comments to help your review.  I'll try to practice [the branching flow](https://github.com/solvcon/devenv/wiki/Branching-flow).

**Comment by yungyuc** [comment]
Assuming we do not have immediate comments, I am going to merge.

**Review by tai271828 — COMMENTED** [review]
The overall looks good to me. Thanks for merging.

**Pull request merged**